runtime/dancebox/tests/common/mod.rs

```diff
@@ -288,6 +272,53 @@ impl
     }
 }
 
+pub fn default_config() -> pallet_configuration::HostConfiguration {
+    pallet_configuration::HostConfiguration {
+        max_collators: 100,
+        min_orchestrator_collators: 2,
+        max_orchestrator_collators: 2,
+        collators_per_container: 2,
+        full_rotation_period: 24,
+        ..Default::default()
+    }
+}
+
+pub struct ExtBuilder {
+    // endowed accounts with balances
+    balances: Vec<(AccountId, Balance)>,
+    // [collator, amount]
+    collators: Vec<(AccountId, Balance)>,
+    // sudo key
+    sudo: Option<AccountId>,
+    // list of registered para ids: para_id, genesis_data, boot_nodes, block_credits, session_credits
+    para_ids: Vec<ParaRegistrationParams>,
+    // configuration to apply
+    config: pallet_configuration::HostConfiguration,
+    safe_xcm_version: Option<u32>,
+    own_para_id: Option<ParaId>,
+}
+
+impl Default for ExtBuilder {
+    fn default() -> Self {
```

Comment:
Interesting that we did not have a default :(

Reply:
We did, but it had no collators, so any tests that needed collators had to explicitly define them.